acceptance/openstack/imageservice/v2/images_test.go

```diff
@@ -148,7 +148,9 @@ func TestImagesFilterByTags(t *testing.T) {
 	defer DeleteImage(t, client, image)
 
 	listOpts := images.ListOpts{
-		Tags: []string{"foo", "bar"},
+		Tags:            []string{"foo", "bar"},
+		ContainerFormat: "bare",
```

Comment:
Per https://developer.openstack.org/api-ref/image/v2/#show-images `the in operator` both these look like they could be more complicated, but this should work as is for single value matching.

Reply:
Right. I figured the easiest thing might be to keep these as string as to allow people to pass in non-single-value strings.